ui/src/app/views/concept-homepage/component.html

```diff
@@ -48,16 +48,21 @@
   <ng-template #chartBox>
     <div class="chart-box lower-box">
       <div class="chart-selector-tabs">
-        <div *ngFor="let domain of conceptDomainList" class="chart-selector-tab">
-          <button (click)="selectDomain(domain.domain)" class="domain-selector-button btn btn-link nav-link nav-item" [class.active]="domain.domain === selectedDomain" [disabled]="searchLoading">
+        <div *ngFor="let domain of conceptDomainCounts" class="chart-selector-tab">
+          <button (click)="selectDomain(domain.domain)" class="domain-selector-button btn btn-link nav-link nav-item" [class.active]="domain.domain === selectedDomain" [disabled]="searchLoading || !completedDomainSearches.includes(domain.domain)">
             <div class="chart-selector-header">
               {{domain.name}}
             </div>
-            <!-- TODO (RW-1277): Make this the counts returned from search. -->
-            <div>{{domain.allConceptCount}}</div>
+            <div>{{domain.concept_count}}</div>
           </button>
         </div>
       </div>
+      <div class="filter-options">
+        Filter by vocabulary:
+        <clr-checkbox class="filter-checkbox" *ngFor="let vocabulary of vocabularies" (click)="filterList()" [(clrChecked)]="vocabulary.selected">
```

Comment:
Huh, what's the significance of `[(clrChecked)]` ? vs just `[]` or just `()`

Reply:
Ooooh! Teaching time, cause I just learned this recently-ish! In angular `[]` is an input binding, and `()` is an output binding. So `[(clrChecked)]` is a two way binding to the variable, such that it defaults to the state passed in, and then changes to the state of the checkbox component are propegated back out to the model variable. Similar to why we do `[(ngModel)]` vs `[inputVariable]` or `(outputHandler)`